src/components/Menus/Sidebar/SettingsPanel/SettingsPanel.scss

```
@@ -0,0 +1,14 @@
+@use "../../../../spacing" as *;
+@use "../../../../font-size" as *;
+@import "../../../../colours.scss";
+
+.settings-panel {
+  color: $rpf-text-primary;
```

needs a font family set too
```suggestion
  color: $rpf-text-primary;
  font-family: sans-serif;
```